### PR TITLE
[service discovery] marathon: Expose containerPort as __meta label for marathon_sd_config

### PR DESCRIPTION
**Pull request by liguozhong**
https://github.com/prometheus/prometheus/issues/4310
Signed-off-by: fuling <fuling.lgz@alibaba-inc.com>

### CODE REVIEW

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -896,6 +896,7 @@ create a target group for every app that has at least one healthy task.
 The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_marathon_app`: the name of the app (with slashes replaced by dashes)
+* `__meta_marathon_container_port`: the docker container port
```
Comment (brian-brazil) [inline docs/configuration/configuration.md:899]: Docker is in the following line.

Is is a Docker or any container management software?

**Comment by brian-brazil** [comment]
This seems redundant, it's already in the address.

**Comment by liguozhong** [comment]
> This seems redundant, it's already in the address.

#4310 This issue is not closed yet

**Comment by liguozhong** [comment]
> This seems redundant, it's already in the address.

I went to complete other issues, the list of prometheus issues is too long, and more contributors are needed

**Pull request closed without merging**